### PR TITLE
Add and update troubleshooting commands

### PR DESCRIPTION
**Pull request by vrabbi**
Signed-off-by: Scott Rosenberg <scott@tera-sky.com>

## What this PR does / why we need it
This updates the process of getting logs from the infra provider controller in a simpler manner then the current documentation.
This also adds commands in an additional section at the bottom that are helpful when debugging clusters

## Details for the Release Notes (PLEASE PROVIDE)
Update Troubleshooting Docs
```release-note

```

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: docs/site/content/docs/latest/tanzu-debugging-tips.md [hunk 1]
```diff
@@ -31,27 +31,21 @@ The previous shows a list of active containers in docker (the output for your
...
-    * Azure: `capz-controller-manager` / `capz-system`
-    * docker: `capd-controller-manager` / `capd-system`
-    * vSphere: `capv-controller-manager` / `capv-system`
-
-1. Retrieve the log for that controller to verify why it is unable to complete
-   the management cluster provision:
-
-    ```shell
-    docker exec <CONTAINDER_ID> kubectl logs capa-controller-manager-<UNIQUE_ID> -n capa-system -c manager --kubeconfig /etc/kubernetes/admin.conf
+    # For AWS
```
Comment (stmcginnis) [inline docs/site/content/docs/latest/tanzu-debugging-tips.md:37]: I don't think the mdlinter will like using top level headers inside the bullet list. You may need to change this to `##` or even `###`. Might want that for formatting purposes too.

File: docs/site/content/docs/latest/tanzu-debugging-tips.md [hunk 2]
```diff
@@ -101,3 +95,24 @@ management cluster.
     docker stop 2f505b8b0c8a && docker rm 2f505b8b0c8a
     docker stop 3dd82b26ac04 && docker rm 3dd82b26ac04
     ```
+## Aditional helpful commands for debugging clusters
```
Comment (stmcginnis) [inline docs/site/content/docs/latest/tanzu-debugging-tips.md:98]: Will need a blank line between the closing codeblock and this header.

File: docs/site/content/docs/latest/tanzu-debugging-tips.md [hunk 3]
```diff
@@ -101,3 +95,24 @@ management cluster.
...
+1. Get all Cluster API related objects:
+```sh
```
Comment (stmcginnis) [inline docs/site/content/docs/latest/tanzu-debugging-tips.md:100]: Same between all of these. Will need to also indent the codeblocks so they line up with the first letter of the text of the bullet (so add three leading spaces to account for the `1. `).

**Review by stmcginnis — COMMENTED** [review]
This looks great. Just some minor formatting issues that the markdown linter is going to complain about. That can be tested locally by running `./hack/check-mdlint.sh`.

File: docs/site/content/docs/latest/tanzu-debugging-tips.md (hunk 1 above)
Comment (vrabbi) [reply docs/site/content/docs/latest/tanzu-debugging-tips.md]: I've changed it to ##. mdlint didnt complain about a single # but always better to play it safe.

File: docs/site/content/docs/latest/tanzu-debugging-tips.md [hunk 4]
```diff
@@ -76,7 +70,7 @@ management cluster.
     tanzu management-cluster delete <YOUR-CLUSTER-NAME>
     ```
 
-1. Next, if the bootstrap cluster still exists, delete it:
+2. Next, if the bootstrap cluster still exists, delete it:
```
[the remark this answers is not in the document]
Comment (vrabbi) [reply docs/site/content/docs/latest/tanzu-debugging-tips.md]: changed it back to all 1s

File: docs/site/content/docs/latest/tanzu-debugging-tips.md (hunk 2 above)
Comment (vrabbi) [reply docs/site/content/docs/latest/tanzu-debugging-tips.md:99]: done

File: docs/site/content/docs/latest/tanzu-debugging-tips.md (hunk 3 above)
Comment (vrabbi) [reply docs/site/content/docs/latest/tanzu-debugging-tips.md]: done

**Review by stmcginnis — APPROVED** [review]
Looks great, thanks!

**Pull request merged**